nextcord/message.py

```diff
@@ -820,6 +868,11 @@ def __init__(
             _component_factory(d) for d in data.get("components", [])
         ]
         self._background_tasks: Set[asyncio.Task[None]] = set()
+        self._call: MessageCallPayload | None = data.get("call")
+        if self._call is not None:
+            self.call: Optional[MessageCall] = MessageCall(state=state, data=self._call)
+        else:
```

Comment:
This could be shortened to:
```py
if call_data := data.get("call"):
    self.call: Optional[MessageCall] = MessageCall(state=state, data=call_data)
else:
    self.call: Optional[MessageCall] = None
```
The `_call` attribute is useless since u can access data from [MessageCall](https://github.com/nextcord/nextcord/blob/35269c26c30c7bf813a755b74eec5addb97f1e39/nextcord/message.py#L649)